container/shim/src/fetchers/lassie.js

```diff
@@ -105,13 +105,13 @@ export async function respondFromLassie(req, res, { cidObj, format }) {
     } else {
       res.set("Cache-Control", "public, max-age=29030400, immutable");
     }
-    const carStream = lassieRes.body.pipe(metricsStream);
+    pipeline(lassieRes.body, metricsStream, (err) => {});
```

Comment:
Just double checking, do we want to not log errors here?

Reply:
We do. That's something I can implement post merge

Reply:
After testing, errors are caught by the below `catch` block. An empty catch handler seems fine here?

Reply:
yeah, that's the point of the empty callback - those errors end up coming back through the async iterator from js-car anyway; but now you have multiple options! not confusing at all

Reply:
Yep i added one. Also worth noting that if `pipeline` throws an error if the last argument isn't a callback, so you have to pass one.

Reply:
fwiw you could also just grab the version from `'node:stream/promises'` and drop the returned Promise, no callback needed. This is more explicit I think though.